examples/next/app-router/tsconfig.json

```diff
@@ -13,8 +13,8 @@
     "noEmit": true,
     "incremental": true,
     "esModuleInterop": true,
-    "module": "esnext",
-    "moduleResolution": "node",
+    "module": "ESNext",
+    "moduleResolution": "Bundler",
```

Comment:
In order to use package exports in TypeScript, you must set `moduleResolution` to `Node16`, `NodeNext`, or `Bundler`. Since we don't want to enforce strict module resolution with file extensions, etc. on our users, we use `Bundler` since the other two require that strict behavior.